src/pyFAI/integrator/fiber.py

```diff
@@ -26,44 +26,131 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-__author__ = "Edgar GUTIERREZ FERNANDEZ "
-__contact__ = "edgar.gutierrez-fernandez@ESRF.eu"
+__author__ = "Edgar Gutiérrez Fernández "
+__contact__ = "edgar.gutierrez-fernandez@esr.fr"
 __license__ = "MIT"
 __copyright__ = "European Synchrotron Radiation Facility, Grenoble, France"
-__date__ = "09/10/2024"
+__date__ = "24/10/2024"
 __status__ = "stable"
 __docformat__ = 'restructuredtext'
 
 import logging
 logger = logging.getLogger(__name__)
 import numpy
+import matplotlib.pyplot as plt
```

Comment:
It is not a good idea to import `matplotlib` or `gui` in a computationnal module